src/PayU/Alu/Card.php

```diff
@@ -35,21 +35,41 @@ class Card
 
     /** @var bool */
     private $enableTokenCreation;
+    /**
+     * @var int|null
+     */
+    private $timeSpentTypingNumber;
```

Comment:
Adding this feature should have been implemented separately.
It wasn't added until now intentionally and it's not related to to v3/v4 api changes.